keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandlerTests.kt

```diff
@@ -393,17 +393,31 @@ internal class SecurityGroupHandlerTests : JUnit5Minutests {
       }
     }
 
-    context("only inbound rule points to a non-existent security group") {
+    context("one of the three inbound rules points to a non-existent security group") {
 
       deriveFixture {
         copy(
           cloudDriverResponse1 = cloudDriverResponse1.copy(
             inboundRules = setOf(
```

Comment:
If you want to go back to it later, this should work:
```suggestion
            inboundRules = sortedSetOf(
              compareBy { it.securityGroup!!.id },
```
...and then you change the IDs down below to make sure they have the order you like.